src/engine/controls/loopingcontrol.h

```diff
@@ -169,6 +187,8 @@ class LoopingControl : public EngineControl {
     ControlObject* m_pCOLoopMove;
     QList<LoopMoveControl*> m_loopMoves;
 
+    CuePointer m_pCue;
```

Comment:
Is this member actually used?

Reply:
No, leftover from an earlier approach to track the saved loop state. Removed.